dashboard-app/src/navigation/Navbar/index.tsx

```diff
@@ -28,30 +34,25 @@ const Title = styled.p`
 `;
 
 
-const pageFromRoute = (url: string): string => {
-  const routes: Dictionary<string> = {
-    '/activity': 'activity',
-    '/time': 'time',
-    '/volunteer': 'volunteer',
-    '/': '/',
-  };
-
-  return routes[url];
-};
-
+/**
+ * Component
+ */
 const Navbar: React.FunctionComponent<Props> = (props) => {
-  const isLoggedIn = Boolean(pageFromRoute(props.pathname));
-  const active = pageFromRoute(props.pathname);
+  const currentPage = Pages.matchPath(props.pathname);
+  const isLoggedIn = currentPage.protected;
+  const links = Pages.getProtected().map((page) => ({
+    to: page.url,
+    content: page.title,
+    active: page.url === currentPage.url,
+  }));
 
   return(
   <PaddedRow middle="xs" between="xs">
     <Col xs={6} lg={4}>
       <Title>TWINE</Title>
     </Col>
     <Col xs={6} lg={4}>
-    {isLoggedIn &&
-      <Navigation active={active}/>
-    }
+      { isLoggedIn && <NavLinks links={links} withLogout={isLoggedIn}/> }
```

Comment:
Is there a page where we want the navlinks showing but not the logout link?

Reply:
The designs have navbar links on the login page

Reply:
The designs were not meant to be a UX design in this sense. And I think we should be careful coding in specifics that are shown. 

It doesn’t matter too much for this as it’s a small change, but worth thinking about as we develop the rest of the dashboard.